pkg/bandwidth/bandwidth.go

```
@@ -74,6 +76,82 @@ func ProbeBandwidthManager() {
 	}
 }
 
+func setBaselineSysctls() error {
+	// Ensure interger type sysctls are no smaller than our baseline settings
+	baseIntSettings := map[string]int64{
+		"net.core.netdev_max_backlog":  1000,
+		"net.core.somaxconn":           4096,
+		"net.ipv4.tcp_max_syn_backlog": 4096,
+	}
+
+	for name, value := range baseIntSettings {
+		currentValue, err := sysctl.ReadInt(name)
+		if err != nil {
+			return fmt.Errorf("read sysctl %s failed", name)
```

The original error is lost here

---

Done, thanks!